reference/data-node-bootstrapping.md

```diff
@@ -0,0 +1,130 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+In order for an access node to be able to use a data node it is
+necessary to ensure that it is properly setup to work as a data
+node. This is called *bootstrapping* the data node.
+
+When an access node adds a new data node to the distributed database
+it either bootstraps the data node or verifies that the data node is
+properly bootstrapped.
+
+A new data node is added to the access node using the `add_data_node`
+command. The default database to bootstrap on the data node is then
+given by the same name, that is if you are connected to the `my_data`
+database on the access node, it will be `my_data` on the data node as
+well. If you do not want to use the same database on the data node as
```

Comment:
I was confused by the formation, since it is not clear `the same name` as what, and it is unclear that `my_data` is an example.
So here is my suggestion:

> The default database to bootstrap has the same name as the access node database, i.e., the connected database from where `add_data_node` is called. For example, the name of the database is `my_test_db` on the access node and `add_data_node` is called, then a database will be bootstrap with name `my_test_db`. In the case of the database `postgres`, which is common starting point, `postgres` database will be bootstrap on the data node.

Reply:
Reworded.